src/modules/home/components/property/property-card.tsx

```diff
@@ -14,16 +16,37 @@ import './swiper.css';
 
 import { EffectFade, Navigation, Pagination } from 'swiper/modules';
 
-export function ApartmentCard(apartment: Accommodation) {
+export function ApartmentCard({ id, name, location, pricePerNight, images }: Accommodation) {
+  const [isLiked, setIsLiked] = useState(false);
+  const navidate = useNavigate();
+  useEffect(() => {
+    const likedStatus = localStorage.getItem(`liked-${id}`);
```

Comment:
Is there a backend integration for the wishlist?

Reply:
changed